website/docs/getting-started/adoption.md

```diff
@@ -1,9 +0,0 @@
----
-title: Adoption
-description: This is the (missing) long description of the component, that will come from the frontmatter attributes
-order: 103
```

Comment:
Shouldn't we just add `hidden: true` to these pages instead of removing them?

Reply:
Admittedly I did forget that was an option. I'm inclined to remove them, but for now I've marked them as hidden.